cpp/src/arrow/filesystem/s3fs.h

```diff
@@ -155,7 +155,7 @@ struct ARROW_EXPORT S3Options {
   /// Whether OutputStream writes will be issued in the background, without blocking.
   bool background_writes = true;
 
-  /// Whether to allow creation of buckets
+  /// Whether to disallow creation of buckets
```

Comment:
This change seems spurious, could you revert it?

Reply:
My bad, just reverted